frontend/libs/studio-components/src/components/StudioProfileMenu/StudioProfileMenu.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { type ReactNode, type ReactElement, useState } from 'react';
+import classes from './StudioProfileMenu.module.css';
+import { Divider, DropdownMenu } from '@digdir/designsystemet-react';
```

Comment:
Should we use StudioDropdownMenu here? 🤔

Reply:
Nope. Can't use it as the implementation of it does not work correctly with this use case 😄

Reply:
I see 😄